install.ps1

```diff
@@ -6,14 +6,15 @@ $ErrorActionPreference = 'Stop'
 
 if ($v) {
   $Version = "v${v}"
-}
-if ($Args.Length -eq 1) {
-  $Version = $Args.Get(0)
+} elseif ($Args.Length -eq 1) {
```

Comment:
I don't think this should be an elseif because now `$v` being set takes priority, but it should be the other way around.

Reply:
Okay got it 👍🏻